ui/app/mainui/activitycenter/controls/CommunityBadge.qml

```diff
@@ -86,20 +81,12 @@ Badge {
                 color: Theme.palette.baseColor1
             }
 
-            StyledTextEdit {
-                Layout.maximumWidth: 300
+            StatusLinkText {
+                Layout.fillWidth: true
                 Layout.alignment: Qt.AlignVCenter
                 visible: root.channelName.length > 0
-                text: Utils.getLinkStyle("#" + root.channelName, hoveredLink, Theme.palette.baseColor1)
-                readOnly: true
-                textFormat: Text.RichText
-                clip: true
-                color: Theme.palette.baseColor1
-                font.pixelSize: 13
-                font.weight: Font.Medium
-                onLinkActivated: {
-                    root.channelNameClicked()
-                }
+                text: "#" + root.channelName
```

Comment:
Where has `Utils.getLinkStyle()` disappeared?

Reply:
Yes, now it's useless :)

Reply:
https://user-images.githubusercontent.com/2522130/219346744-faf93b1d-5764-4aa0-b79b-310d1a2e92b2.mov

Reply:
The link should be blue when hovered, it's not useless 😊

Reply:
<img width="537" alt="image" src="https://user-images.githubusercontent.com/2522130/219373550-23512c0d-637c-4b79-900a-ba8499c94e50.png">
It was overriden, but i can add blue on hover in StatusQ component

Reply:
Yeah, you can add a `linkColor` value to `StatusLinkText`

Reply:
<img width="545" alt="Screenshot 2023-02-17 at 12 31 08" src="https://user-images.githubusercontent.com/2522130/219594641-bf61e278-0f5a-4412-b9bb-507f9330108a.png">
@caybro fixed

Reply:
Cool, thx!